CHANGELOG.md

```diff
@@ -41,12 +41,6 @@
 - Dependencies on `fivetran/fivetran_utils` have been upgraded, previously `[">=0.3.0", "<0.4.0"]` now `[">=0.4.0", "<0.5.0"]`.
 
 [PR #40](https://github.com/fivetran/dbt_shopify_source/pull/40) includes the following breaking changes:
-- In the following tables, `*_timestamp` columns have been renamed to `*_at` for consistency:
-  - `stg_shopify__customer`
-  - `stg_shopify__order`
-  - `stg_shopify__product_variant`
-  - `stg_shopify__product`
-  - `stg_shopify__transaction`
```

Comment:
Was there a reason we just changed these tables to the `timestamp` format? I see some others still in the `_at` format. https://github.com/fivetran/dbt_shopify_source/blob/feature/package-revamp/models/stg_shopify__refund.sql

Reply:
not that i can tell, it seems like a stylistic choice Dylan made for the most pertinent timestamp columns

Reply:
Ah got it!